src/components/Tokens/TokenTable/NetworkFilter.tsx

```diff
@@ -101,7 +100,7 @@ export default function NetworkFilter() {
 
   return (
     <StyledMenu ref={node}>
-      <FilterOption onClick={toggleMenu} aria-label={`networkFilter`} active={open}>
+      <FilterOption width="156px" onClick={toggleMenu} aria-label={`networkFilter`} active={open}>
```

Comment:
nit: should extend FilterOption for this and then set the width there, rather than passing as a prop

Reply:
the issue is the first and last aren't actually centered, there's a little more spacing above/below in the design